src/test/input.ts

```diff
@@ -0,0 +1,42 @@
+export const user1 = `#graphql
```

Comment:
esse nome `user1` não é muito claro, talvez `createUserMutation` ou algo do tipo?
A propósito, você pode utilizar as variáveis no graphql ao invés e forçar o valor no texto:
```
const mutation = `#graphql
mutation createUser(data: UserInput!) {
  createUser(data: $data) {
    id
    name
    email
    birthDate
  }
}

    const response = await axios(url, {
      query: createUserMutation,
      variables: {
        data: {
          name: 'eu',
          email: 'eu@gmail.com',
          birthDate: '01/01/1971',
          password: '123abc',
        },
      },
    });
```

Com isso, você consegue reaproveitar o texto da mutation para diferentes cenários de teste